Sources/Paywalls/PaywallData.swift

```diff
@@ -99,8 +99,12 @@ extension PaywallData {
     }
 
     /// - Returns: ``PaywallData/LocalizedConfiguration-swift.struct`` for the given `Locale`, if found.
-    public func config(for locale: Locale) -> LocalizedConfiguration? {
-        return self.localization[locale.identifier]
+    /// - Note: this allows searching by `Locale` with only language code and missing region (like `en`, `es`, etc).
+    public func config(for requiredLocale: Locale) -> LocalizedConfiguration? {
+        self.localization[requiredLocale.identifier] ??
+        self.localization.first { locale, _ in
```

Comment:
@joshdholtz we'll need to implement the same logic in the backend when requesting a specific locale (or did we decide we weren't gonna do that?)

Reply:
@NachoSoto We decided against that for now so nothing to do there 😊